config/routes.rb

```diff
@@ -668,6 +653,66 @@ def route_actions_hash
     actions: LEGACY_CRUD_ACTIONS - [:destroy] + [:show_past]
   )
 
+  # ----- Herbaria: standard actions -------------------------------------------
+  namespace :herbaria do
+    resources :curator_requests, only: [:new, :create]
+    resources :curators, only: [:create, :destroy], id: /\d+/
+    resources :merges, only: [:create]
+    resources :nexts, only: [:show], id: /\d+/
+  end
+  resources :herbaria, id: /\d+/
+  # Herbaria: standard redirects of Herbarium legacy actions
+  redirect_legacy_actions(
+    old_controller: "herbarium", new_controller: "herbaria",
+    actions: LEGACY_CRUD_ACTIONS - [:controller, :index, :show_past]
+  )
+  # Herbaria: non-standard redirects of legacy Herbarium actions
+  # Rails routes currently accept only template tokens
+  # rubocop:disable Style/FormatStringToken
+  get("/herbarium/herbarium_search", to: redirect(path: "herbaria"))
+  get("/herbarium/index", to: redirect(path: "herbaria"))
+  get("/herbarium/list_herbaria", to: redirect(path: "herbaria?flavor=all"))
+  get("/herbarium/request_to_be_curator/:id",
+      to: redirect(path: "herbaria/curator_requests/new?id=%{id}"))
+
+  # Herbaria: complicated redirects of legacy Herbarium actions
+  # Actions needing two routes in order to successfully redirect
+  #
+  # The next two routes combine to redirect
+  #   GET herbarium/delete_curator
+  #   DELETE herbaria/curators
+  # The "match" redirects
+  #   GET("/herbarium/delete_curator/nnn?user=uuu") and
+  #   POST("/herbarium/delete_curator/nnn?user=uuu")
+  # to
+  #   GET("/herbaria/curators/nnn?user=uuu")
+  # which would throw: No route matches [GET] "/herbaria/curators/nnnnn"
+  # absent the following get
+  match("/herbarium/delete_curator/:id",
+        to: redirect(path: "/herbaria/curators/%{id}"),
+        via: [:get, :post])
+  get("/herbaria/curators/:id", to: "herbaria/curators#destroy", id: /\d+/)
+
+  # The next post and get combine to redirect the legacy
+  #   POST /herbarium/request_to_be_curator to
+  #   POST herbaria/curator_requests#create
+  post("/herbarium/request_to_be_curator/:id",
+       to: redirect(path: "/herbaria/curator_requests?id=%{id}"))
+  get("/herbaria/curator_requests",
+      to: "herbaria/curator_requests#create", id: /\d+/)
+
+  # The next post and get combine to redirect
+  #   POST /herbarium/show_herbarium/:id to
+  #   POST herbaria/curators#create
+  post("/herbarium/show_herbarium", to: redirect(path: "herbaria/curators"))
+  get("/herbaria/curators", to: "herbaria/curators#create", id: /\d+/)
```

Comment:
### Should I also remove these redirect routes?
Although they work, they raise issues similar to the routes removed in [Commit #0069792](https://github.com/MushroomObserver/mushroom-observer/pull/740/commits/006979285243a1086b89f880ce590864123b3272): using a GET to write data, unlikely that there are links that point to the legacy actions, and even if such links existed, it's unlikely anyone would complain.